pod/enrichment/static/js/enrichment.js

```diff
@@ -41,155 +63,188 @@ var ajaxfail = function (data) {
       gettext("The form could not be recovered."),
     "alert-danger"
   );
-  $("form.get_form").show();
+  document.querySelector("form.get_form").style.display = "block";
+
   show_form("");
 };
 
-$(document).on("click", "#cancel_enrichment", function () {
-  $("form.get_form").show();
+document.addEventListener("click", (e) => {
+  if (e.target.id != "cancel_enrichment") return;
+  document.querySelectorAll("form.get_form").style.display = "block";
```

Comment:
Ce script plante quand on clique sur le bouton "annuler" on a l'erreur 
`TypeError: undefined is not an object (evaluating 'document.querySelectorAll("form.get_form").style.display = "block"')`

Reply:
il manque surement un "for each"